thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -2255,8 +2255,10 @@ class CLookupJoinActivityBase : public CInMemJoinBase<HTHELPER, IHThorHashJoinAr
             }
             virtual bool spill(bool critical) // called from OOM callback
             {
+                if (!channelCollector->spill(critical))
+                    return false;
```

Comment:
There are concurrent calls to putRow() (at least in a multi-channels setup). All channels are determining hashing from local to channel, so any may call any other channel's CChannelDistributor::putRow() (via CChannelDistributor::process())